README.md

```diff
@@ -36,18 +36,17 @@ git checkout master
 
 2. Set up the Engine development environment: see [the wiki page](https://github.com/flutter/flutter/wiki/Setting-up-the-Engine-development-environment).
 
-3. macOS only: in **the engine repo**, add the prototype remote and switch to the prototype branch:
+3. Apply patches depending on platform.
```

Comment:
Could these be sub-items? Something like this, but double-check my formatting

````
3. Apply patches depending on platform.

   * macOS only: in **the engine repo**, apply this playground's patch:

     ```bash
     git apply /path/to/playground/patches/macos/001-Add-multi-view-Flutter-macOS-APIs.patch
     ```

   * Windows only: in **the engine repo**, apply this playground's patch:

     ```bash
     git apply /path/to/playground/patches/windows/001-Add-multi-view-Flutter-Windows-C++-APIs.patch
     ```
````

Reply:
(I applied this suggestion before merging)